CONTRIBUTING.md

```diff
@@ -39,22 +34,16 @@ To contribute data to the Mobility Database catalogs, please follow these steps:
 11. When your contribution is ready, convert your pull request from draft to ready for review and request a review from a team member at Mobility Data. Not that if you need to modify your contribution after this step, you will be asked to convert your pull request back to draft.
 12. Once your pull request is converted to ready for review and that all the checks have passed, we will approve and merge it.
 
-### Contribute data
-
 #### Add a GTFS Schedule source
 The easiest way to add a GTFS Schedule source is to use the operation `tools.operations.add_gtfs_schedule_source` through the Python interpreter or in your scripts. This operation makes sure the information provided is correct and will pass our tests. Provide the information about your source in the operation function to add your source.
 
-Note that you must pass an `api_key_parameter_value` if your source has `authentication_type = 1` or `authentication_type = 2`. The `api_key_parameter_value` will not be stored and is used only for testing before the source is added to the database..
+If your GTFS Schedule source requires API authentication, please use the [form](https://database.mobilitydata.org/update-a-data-source) instead of the PR process so we can generate our own API credentials for the source.
 
 ```python
 >>> add_gtfs_schedule_source(
         provider=$YOUR_SOURCE_PROVIDER_NAME,
         country_code=$YOUR_SOURCE_COUNTRY_CODE,
         direct_download_url=$YOUR_SOURCE_DIRECT_DOWNLOAD_URL,
-        authentication_type=$OPTIONAL_AUTHENTICATION_TYPE,
-        authentication_info_url=$CONDITIONALLY_REQUIRED_AUTHENTICATION_INFO_URL,
-        api_key_parameter_name=$CONDITIONALLY_REQUIRED_API_KEY_PARAMETER_NAME,
-        api_key_parameter_value=$NOT_STORED_API_KEY_PARAMETER_VALUE,
```

Comment:
Oh I see where you're coming from here, but I would not remove those parameters. Specifically, anyone adding sources through our repo can add those with `authentication_type` 1 or 2. Also, the parameters won't be remove from the operation.

I'd rather change the sentence above to emphasize that sources with `authentication_type` 1 or 2 can't be added from forks, but those with `authentication_type=0` can.